guillotina_elasticsearch/utility.py

```diff
@@ -228,7 +230,8 @@ def _get_items_from_result(self, container, request, result):
             items.append(data)
         return items
 
-    async def query(
+
+    async def search(
```

Comment:
I'm not sure about renaming query->search because this would break the current behavior of endpoint @search (that allows raw elastic queries)

Reply:
Yes, we use query right now a lot and this would break us.

Reply:
Default catalog implementation on Guillotina master is that query calls search, and search is the raw query. We need to clarify this situation. @vangheem what do you suggest? personally I don't care if we change pgcatalog to switch query <-> search but we will need to change lot of code that calls query with the parser information.

Reply:
Maybe we should remove ES 7 enforce from guillotina_elasticsearch and switch to master (Isee that you guys are using 3.3.x)

Reply:
+1. I don't know if it would be possible to support both ES 6 and 7

Reply:
I think ideally, we see how feasible es 6/7 support is. Then, we move this to master with it.

Then, I'll still use this branch until we upgrade.